_spinnaker_user_guides/kayenta.md

```diff
@@ -0,0 +1,348 @@
+---
+layout: post
+title: Automatic Canary Analysis (Kayenta)
+order: 74
+published: True
+redirect_from:
+  - /user-guides/deploying/
+  - /user_guides/deploying/
+  - /spinnaker_user_guides/deploying/
```

Comment:
Should this also include `/spinnaker-user-guides/depolying`?

Reply:
Nice catch - this is actually just all wrong.  Should be `kayenta` instead of `deploying`.